docs/GettingStarted.md

```diff
@@ -66,7 +69,8 @@ $ cmake -G Ninja ../llvm \
     -DLLVM_ENABLE_PROJECTS="mlir" \
     -DLLVM_TARGETS_TO_BUILD="X86;RISCV" \
     -DLLVM_ENABLE_ASSERTIONS=ON \
-    -DCMAKE_BUILD_TYPE=DEBUG
+    -DCMAKE_BUILD_TYPE=DEBUG \
```

Comment:
I'm curios if we can also recommend RelWithDebInfo by default. I believe many of us use RelWithDebInfo normally. It would make linking so much easier (probably it's feasible to compile CIRCT on laptop even without lld).

Reply:
My experience is that I have to keep both builds around. It really depends on if you plan/need to use a lldb/gdb. I find I can't get much done with `RelWithDebInfo` in such an environment. However, `RelWithDebInfo` is almost always sufficient. I think suggesting `RelWithDebInfo` is probably fine.